wsd/FileServer.cpp

```diff
@@ -507,7 +507,7 @@ void FileServerRequestHandler::handleRequest(const HTTPRequest& request,
 
         if (requestUri.getPath().find("browser/" COOLWSD_VERSION_HASH "/") == std::string::npos)
         {
-            LOG_WRN("client - server version mismatch, disabling browser cache. Expected: " COOLWSD_VERSION_HASH);
+            LOG_WRN("client - server version mismatch, disabling browser cache. Expected: " COOLWSD_VERSION_HASH "; Actual URI path with version hash: " + requestUri.getPath());
```

Comment:
Best to use clang-format, which would break this long line, to make the code consistent and easier for other without scrolling side-ways (when diffing side-by-side, or 3-way merging, we have less than half or 1/3rd the screen width).

Reply:
Also, please use `<<` in logs instead of `+`.

Reply:
Thanks a bunch for the hint, hopefully it's better now.